src/core/Microsoft.PowerToys.Settings.UI.Runner/Program.cs

```diff
@@ -26,7 +26,7 @@ public static void Main(string[] args)
 
                 if (args.Length > 3)
                 {
-                    if (args[4] == "true")
+                    if (args[3] == "true")
```

Comment:
the fourth argument should be the elevation status

Reply:
line 291: https://github.com/microsoft/PowerToys/blob/dev/build-features/src/runner/settings_window.cpp
theme name is not initialized. Resulting in less num of args sent.
![image](https://user-images.githubusercontent.com/58791731/81137232-f6bd3600-8f12-11ea-89ad-ed9097213b91.png)

Assign default value to fix it:
![image](https://user-images.githubusercontent.com/58791731/81137256-063c7f00-8f13-11ea-9167-10a0eb1fb9de.png)

Reply:
no changes should be made on this line.

Reply:
Reverted the args change and initialised the settings_theme variable.